pandas/plotting/_timeseries.py

```diff
@@ -22,16 +22,22 @@
 
 def tsplot(series, plotf, ax=None, **kwargs):
     """
-    Plots a Series on the given Matplotlib axes or the current axes
+    Plots a Series on the given Matplotlib axes or the current axis.
```

Comment:
I *think* that matplotlib uses the phrase "current axes", so the old way was correct here. "axis" is for the "x" or "y" axis of an axes :)

Good you change that back here and on line 33?